data/json/mapgen/mansion_boarded.json

```diff
@@ -1018,7 +1018,7 @@
         ",": { "item": "softdrugs", "chance": 40 },
         "?": { "item": "dresser_servant", "chance": 45 }
       },
-      "place_monster": [ { "group": "GROUP_MANSION_ESCAPE", "x": [ 18, 23 ], "y": [ 8, 23 ], "chance": 65, "repeat": [ 3, 5 ] } ]
+      "place_monster": [ { "group": "GROUP_ZOMBIE", "x": [ 18, 23 ], "y": [ 8, 23 ], "chance": 65, "repeat": [ 3, 5 ] } ]
```

Comment:
All instances of `"GROUP_ZOMBIE"` in this PR are a regression of a previous problem of zombies spawning where they do not make sense to be (firefighters and cops for example), the best group to use here (since the mansion groups are removed in this PR) is the `"GROUP_VANILLA"` since that only has normal, vanilla civilians that would make sense in a residential building.

Reply:
I considered GROUP_VANILLA and GROUP_HOUSE but I'd already changed most of them and didn't feel like it was that big of a deal, I changed from the mansion group bc it doesn't really warrant having a unique group without the ferals

Reply:
Removal of the mansion group is understandable, but yeah, there is an old (minor) problem with specialized monsters appearing where they do not make sense, the zombie group even includes soldier zombies, for general areas like the street of a city or gathering locations this doesn't really matter, but in residential buildings is quite notable when you find enemies that do not really belong there.